frame/staking/src/pallet/impls.rs

```diff
@@ -203,7 +207,12 @@ impl<T: Config> Pallet<T> {
 	/// This will also update the stash lock.
 	pub(crate) fn update_ledger(
 		controller: &T::AccountId,
-		ledger: &StakingLedger<T::AccountId, BalanceOf<T>>,
+		ledger: &StakingLedger<
+			T::AccountId,
+			BalanceOf<T>,
+			ConstU32<MAX_UNLOCKING_CHUNKS>,
```

Comment:
```suggestion
			MaxUnlokcingChunks,
```

Now that you are adding a million type configs to staking, please make this a type as well :D

Reply:
:D done